compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -2199,6 +2208,8 @@ private ParserRuleContext getNextRuleForTypeDescriptor() {
                 return ParserRuleContext.IDENTIFIER;
             case TYPE_DESC_IN_STREAM_TYPE_DESC:
                 return ParserRuleContext.STREAM_TYPE_FIRST_PARAM_RHS;
+            case TYPED_BINDING_PATTERN:
+                return ParserRuleContext.BINDING_PATTERN;
```

Comment:
Do we need this?

Reply:
when there are multiple binding patterns, this will be neat I think

Reply:
no, I missed it out